### ISSUE TITLE
Voice websocket closed: Session is no longer valid

### ISSUE THREAD

**Issue by duwayy** [teamreflex/DiscordPHP#205]
Hello,
Strange problem, I currently launch a php bot (I'm on Ubuntu) to loop a song over voice channel in a screen terminal to keep it running when I close terminal.
It perfectly works, BUT when I'm trying to auto launch screen on server startup, bot connect to channel and does not play anything.

For auto start screen at startup, in my /etc/rc.local I use:
`screen -S discordncs -d -m php /var/www/html/discord/ncsbot.php`
(note that if I launch this command in terminal it will work, bot will play music)

Here is output of screen launched automatically at server startup:

```
[2017-01-31 14:13:49] Voice-197048494987542529.DEBUG: connected to voice websocket [] []
[2017-01-31 14:13:49] Voice-197048494987542529.DEBUG: sending identify {"packet":{"op":0,"d":{"server_id":"197048494987542520","user_id":"275949637423202300","session_id":"11f20e564f4c705978acb247c94c5c57","token":"2a6fbe93d0881510"}}} []
[2017-01-31 14:13:49] Voice-197048494987542529.WARNING: voice websocket closed {"op":4006,"reason":"Session is no longer valid."} []
```

Any idea?

Thanks !

**Comment by Annihilator708** [comment]
`Attempting connection...                                                                                             Task exception was never retrieved                                                                                   future: <Task finished coro=<VoiceClient.poll_voice_ws() done, defined at C:\Python\Python36\lib\site-packages\discord\voice_client.py:265> exception=TimeoutError()>                                                                     Traceback (most recent call last):                                                                                     File "C:\Python\Python36\lib\site-packages\discord\voice_client.py", line 272, in poll_voice_ws                        yield from self.ws.poll_event()                                                                                    File "C:\Python\Python36\lib\site-packages\discord\gateway.py", line 604, in poll_event                                msg = yield from asyncio.wait_for(self.recv(), timeout=30.0, loop=self.loop)                                       File "C:\Python\Python36\lib\asyncio\tasks.py", line 362, in wait_for                                                  raise futures.TimeoutError()                                                                                     concurrent.futures._base.TimeoutError                                                                                Task exception was never retrieved                                                                                   future: <Task finished coro=<VoiceClient.poll_voice_ws() done, defined at C:\Python\Python36\lib\site-packages\discord\voice_client.py:265> exception=TimeoutError()>                                                                     Traceback (most recent call last):                                                                                     File "C:\Python\Python36\lib\site-packages\discord\voice_client.py", line 272, in poll_voice_ws                        yield from self.ws.poll_event()                                                                                    File "C:\Python\Python36\lib\site-packages\discord\gateway.py", line 604, in poll_event                                msg = yield from asyncio.wait_for(self.recv(), timeout=30.0, loop=self.loop)                                       File "C:\Python\Python36\lib\asyncio\tasks.py", line 362, in wait_for                                                  raise futures.TimeoutError()                                                                                     concurrent.futures._base.TimeoutError                                                                                Fatal error on SSL transport                                                                                         protocol: <asyncio.sslproto.SSLProtocol object at 0x06572B50>                                                        transport: <_SelectorSocketTransport closing fd=1412 read=idle write=<idle, bufsize=0>>                              Traceback (most recent call last):                                                                                     File "C:\Python\Python36\lib\asyncio\sslproto.py", line 638, in _process_write_backlog                                 ssldata, offset = self._sslpipe.feed_appdata(data, offset)                                                       AttributeError: 'NoneType' object has no attribute 'feed_appdata'                                                    Traceback (most recent call last):                                                                                     File "C:\Python\Python36\lib\site-packages\discord\gateway.py", line 604, in poll_event                                msg = yield from asyncio.wait_for(self.recv(), timeout=30.0, loop=self.loop)                                       File "C:\Python\Python36\lib\asyncio\tasks.py", line 358, in wait_for                                                  return fut.result()                                                                                                File "C:\Python\Python36\lib\site-packages\websockets\protocol.py", line 319, in recv                                  raise ConnectionClosed(self.close_code, self.close_reason)                                                       websockets.exceptions.ConnectionClosed: WebSocket connection is closed: code = 4006, reason = Session is no longer valid..`

I experience the same

**Comment by Harmon758** [comment]
> Fatal error on SSL transport protocol: <asyncio.sslproto.SSLProtocol object at 0x06572B50> transport: <_SelectorSocketTransport closing fd=1412 read=idle write=<idle, bufsize=0>> Traceback (most recent call last): File "C:\Python\Python36\lib\asyncio\sslproto.py", line 638, in _process_write_backlog ssldata, offset = self._sslpipe.feed_appdata(data, offset) AttributeError: 'NoneType' object has no attribute 'feed_appdata'

@Annihilator708 This was an irrelevant [issue with asyncio](https://bugs.python.org/issue33037) and should be fixed in Python 3.6.5 now.
